### PR TITLE
Add a copyright notice to network.go

### PR DESCRIPTION
**Pull request by yamt**
The notice was copied from other files in the commit
which introduced this file. [1]

Background: I have a copy of this file in my project, [2]
which uses check-copyrights.sh style checking to ensure
every files have a copyright notice.

[1] 218378bc7e484fe70c72d23dd6c9f1a9a612ec64
[2] https://github.com/midonet/midonet-kubernetes

## Todos
- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

File: utils/network.go [hunk 1]
```diff
@@ -1,3 +1,16 @@
+// Copyright 2015 Tigera Inc
```
Comment (caseydavenport) [inline utils/network.go]: Strictly speaking I think this should say 2018

**Comment by caseydavenport** [comment]
@yamt thanks for the PR :)

Just one comment, plus could you sign the CLA to make the bot happy :)

File: utils/network.go (hunk 1 above)
Comment (yamt) [reply utils/network.go]: should it be?  it's old code.

**Comment by yamt** [comment]
@caseydavenport i'm asking my employer if this cla is ok to sign.

File: utils/network.go (hunk 1 above)
Comment (caseydavenport) [reply utils/network.go]: I believe since the file is being updated in 2018 (even just to add the copyright) it should say 2018.

Comment (yamt) [reply utils/network.go]: ok, i bumped the year. thank you.

**Comment by yamt** [comment]
@caseydavenport now i've signed the cla.

**Pull request merged**